server/api/controllers/forms/get-form-by-id.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
```

Comment:
i think we should combine `get-all-forms` with `get-form-by-id` to just 1 controller: `get-forms` where id is optional, if no id given, return all. it will make it much cleaner and it is also the actual implementation of `Model.find()` anyway. Can take a look at other controllers (sermons, announcements, posts etc.) if need more example